mods/tnt/init.lua

```diff
@@ -96,6 +99,9 @@ local function destroy(drops, npos, cid, c_air, c_fire, on_blast_queue, ignore_p
 		on_blast_queue[#on_blast_queue + 1] = {pos = vector.new(npos), on_blast = def.on_blast}
 		return cid
 	elseif def.flammable then
+		on_construct_queue[#on_construct_queue + 1] =
+				{fn = basic_flame_on_construct,
+				pos = vector.new(npos)}
```

Comment:
Yes that's a nice format. Although if sofar is busy i could alter these on merge easily.